tests/scripts/set_psa_test_dependencies.py

```diff
@@ -160,6 +160,13 @@ def systematic_dependencies(file_name, function_name, arguments):
     #pylint: disable=unused-argument
     """List the systematically determined dependency for a test case."""
     deps = set()
+
+    # Run key policy negative tests even if the algorithm to attempt performing
```

Comment:
Do we really care eventually to be closer to what the current code does? The main thing is that there are configurations in which those tests are run. It is just that my feeling is that this kind of heuristic rule will be difficult to maintain and as they don't bring anything in fact in the test coverage better to not have them at all.

Reply:
I was more aggressive on keeping tests enabled in the default configuration. And without this rule, “PSA key policy: raw agreement, wrong algorithm” currently won't run in any configuration due to the lack of FFDH (or any other direct alternative to ECDH), which clinches it for me.

In the long run, I don't want to have any name-based rule. Instead I want to have the ability to declare exceptions through annotations in the `.function` or `.data` file. But adding ad hoc exceptions made it a one-day no-preliminary-design task, whereas an annotation mechanism would take some time to design and implement.

Reply:
ok, pourrais-tu juste ajouter ces explications au commentaire?

Reply:
done